docs/trace/getting-started/getting-started.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="OpenTelemetry.Exporter.Console" Version="[0.4.0-beta, 1.0)" />

```

Comment:
This is to gather feedback - I think the doc/tutorial should use a published version (package reference) instead of a project reference.

Reply:
How does user see this .csproj? The .md file suggests to install nuget using dotnet install right?

Reply:
Currently the .csproj file is not referenced by any doc / code. It is a self-contained project - which I use to gather feedback.

If we think having a self-contained project is useful for the tutorial, we can add CI enforcement or even reference the code from the doc).

Reply:
I do like this idea. In the event that we have more self-contained samples, it might be nice to have the referenced package version for samples in a central location. In working on the grpc instrumentation I noted that the grpc-dotnet project follows this strategy

Example project: https://github.com/grpc/grpc-dotnet/blob/master/examples/Greeter/Client/Client.csproj#L12
Central place for managing package versions: https://github.com/grpc/grpc-dotnet/blob/master/build/dependencies.props

Reply:
@alanwest good suggestion and we should do it across the repo.